controllers/badge.js

```diff
@@ -20,6 +20,19 @@ const getBadges = async (req, res) => {
   }
 }
 
+const getUserBadges = async (req, res) => {
+  try {
+    const result = await badgeQuery.fetchUserBadges(req.params.username)
+    return result.userExists
+      ? res.json({ message: 'User badges returned successfully!', userBadges: result.userBadges })
+      : res.boom.notFound('The user does not exist')
```

Comment:
`No badges found`?

Reply:
Yes I added that, but I have used a mix of normal `if-else` and ternary operator. I have made a custom response message whose value will be dependent upon whether the `userBadges` array is empty or not. This reduced the number of return statements in the same `if` block.